dspace-api/src/main/java/org/dspace/storage/rdbms/DatabaseUtils.java

```diff
@@ -188,7 +188,9 @@ else if(argv[0].equalsIgnoreCase("migrate"))
                     // NOTE: This looks odd, but all we really need to do is ensure the
                     // DatabaseManager auto-initializes. It'll take care of the migration itself.
                     // Asking for our DB Name will ensure DatabaseManager.initialize() is called.
-                    DatabaseManager.getDbName();
+                    Connection connection = dataSource.getConnection();
+                    DatabaseUtils.updateDatabase(dataSource, connection);
+                    connection.close();
```

Comment:
I don't believe this will work. It fails to fully initialize the DatabaseManager via its `initialize()` method...this means that any DB migrations which depend on isOracle / isPostgres settings or `DatabaseManager.findDbKeyword()` (which are set in `initialize()`) will fail when you just run `dspace database migrate` (but those same DB migrations will succeed if you set `db.migration.automatic=true`). 

There are several migrations which utilize `MigrationUtils.dropDBConstraint()` which likely will not function properly with this change. https://github.com/DSpace/DSpace/blob/master/dspace-api/src/main/java/org/dspace/storage/rdbms/MigrationUtils.java#L37

I'm still not understanding your end goal here. In this PR, there seems to be some possible misunderstanding in how the Flyway DB integration actually initializes/upgrades a DSpace database. I don't quite understand what the problem is in the current Flyway DB integration, so it's hard to make a suggestion for how to make the changes you are attempting.